src/octokit-helper.ts

```diff
@@ -0,0 +1,23 @@
+import * as github from '@actions/github'
+import {Octokit} from '@octokit/rest'
+import {getServerApiUrl} from './url-helper'
+
+// Centralize all Octokit references by re-exporting
+export {Octokit} from '@octokit/rest'
+
+export type OctokitOptions = {
+  baseUrl?: string
+  userAgent?: string
```

Comment:
anything set this `userAgent`?

Reply:
Yes the `ref-helper.ts` injects this currently, https://github.com/peter-murray/checkout/blob/server-url-injection/src/ref-helper.ts#L250